x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -37,25 +38,22 @@ declare module 'puppeteer' {
   function launch(options: ReportingLaunchOptions): Promise<puppeteer.Browser>;
 }
 
-type BrowserConfig = CaptureConfig['browser']['chromium'];
-type ViewportConfig = CaptureConfig['viewport'];
+type ViewportConfig = ReportingConfigType['capture']['viewport'];
 
 export class HeadlessChromiumDriverFactory {
-  private binaryPath: string;
-  private captureConfig: CaptureConfig;
-  private browserConfig: BrowserConfig;
+  private binaryPath?: string;
   private userDataDir: string;
   private getChromiumArgs: (viewport: ViewportConfig) => string[];
   private core: ReportingCore;
 
-  constructor(core: ReportingCore, binaryPath: string, logger: LevelLogger) {
-    this.core = core;
-    this.binaryPath = binaryPath;
-    const config = core.getConfig();
```

Comment:
Calling `core.getConfig` can only be done after the setup phase of the Reporting plugin, due to being implemented before https://github.com/elastic/kibana/pull/88981